MAUI/TreeView/selection.md

```diff
@@ -0,0 +1,167 @@
+
+![.NET MAUI TreeView with FullRowSelect](Images/selection/FullRowSelect.png)
+
+## Selected item style
+
+### Selection background
+
+The TreeView allows changing the selection background color for the selected items by using the `SelectionBackgroundColor` property. You can also change the selection background color at runtime.
+
+### Selection foreground
+
+The TreeView allows changing the selection foreground color for the selected items by using the `SelectionForegroundColor` property. You can also change the selection foreground color at runtime.
+
+N> `SelectionForegroundColor` is applicable only for unbound mode.
+
+## Events
+
+### SelectionChanging Event
+
+The `SelectionChanging` event is raised while selecting an item at the execution time. The `ItemSelectionChangingEventArgs` has the following members which provides the information for `SelectionChanging` event:
+
+* `AddedItems`: Gets collection of the underlying data objects where the selection is going to process.
+* `RemovedItems`: Gets collection of the underlying data objects where the selection is going to remove.
+
+You can cancel the selection process within this event by setting the `ItemSelectionChangingEventArgs.Cancel` property to true.
+
+{% tabs %}
+{% highlight c# %}
+treeView.SelectionChanging += TreeView_SelectionChanging;  
+
+private void TreeView_SelectionChanging(object sender, ItemSelectionChangingEventArgs e)
+{
+   if (e.AddedItems.Count > 0 && e.AddedItems[0] == ViewModel.Items[0])
+       e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### SelectionChanged event
+
+The `SelectionChanged` event will occur once selection process has been completed for the selected item in the TreeView. The `ItemSelectionChangedEventArgs` has the following members which provides information for `SelectionChanged` event:
+
+* `AddedItems`: Gets collection of the underlying data objects where the selection has been processed.
+* `RemovedItems`: Gets collection of the underlying data objects where the selection has been removed.
+
+{% tabs %}
+{% highlight c# %}
+treeView.SelectionChanged += TreeView_SelectionChanged;  
+
+private void TreeView_SelectionChanged(object sender, ItemSelectionChangedEventArgs e)
+{
+   treeView.SelectedItems.Clear();
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> `SelectionChanging` and `SelectionChanged` events will be triggered only on UI interactions.
+
+## Key Navigation
+
+The TreeView allows to select the items through keyboard interactions. Behavior of key navigation is explained as follows:
+
+* When the `SelectionMode` is `Single` or `SingleDeselect`, the selected item is highlighted with `FocusBorderColor` around the item while key navigation.
+
+* When the `SelectionMode` is `Multiple` or `Extended`, the `FocusBorderColor` will set to the `CurrentItem`.
+
+## FocusBorderColor
+
+The `FocusBorderColor` property is used to set the border color for the current focused item.The default color is `Color.FromArgb("#000000")`.

```

Comment:
The `FocusBorderColor` property is used to set the border color for the current focused item. The default color is `Color.FromArgb("#000000")`.